CMakeLists.txt

```diff
@@ -57,6 +57,12 @@ IF(ENABLE_LUSTRE)
   # - LUSTRE_STAT
 ENDIF(ENABLE_LUSTRE)
 
+OPTION(ENABLE_HPSS "Enable optimization and features for HPSSFS-FUSE" OFF)
+MESSAGE(STATUS "ENABLE_HPSS: ${ENABLE_HPSS}")
+IF(ENABLE_HPSS)
+  ADD_DEFINITIONS(-DHPSS_SUPPORT)
```

Comment:
Do we need to add any ``FIND_PACKAGE`` here to set include or lib paths?  It doesn't look like we make any library calls, so perhaps just the header is sufficient.